### DISCUSSION TITLE
An efficient versioning system

### DISCUSSION THREAD

**Discussion by Lamparter** [IviriusCommunity/Rebound#71] [Ideas]
@IviriusMain what do you think?
![image](https://github.com/user-attachments/assets/5956dcd3-f5c2-4ad0-8a92-66a5ad97f6ff)

**Comment by IviriusMain** [comment]
I have something similar in Ivirius Text Editor Plus

**Comment by IviriusMain** [comment]
We can use it, sure

**Reply by Lamparter** [reply]
I thought about it and wrote the README and samples on my phone LMAO
